### PR TITLE
Release 2.3 privacy notice

### PR DESCRIPTION
**Pull request by dsarkar**
Internal Tracking ID: [EXPOSUREAPP-7532](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7532)

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -326,10 +326,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '14.6'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.2.1'))
+    .pipe(replace('[ios.current-app-version]', '2.3.0'))
```
Comment (Ein-Tim) [inline gulpfile.js:329]: Latest iOS 2.3 RC is [2.3.2-RC2](https://github.com/corona-warn-app/cwa-app-ios/releases/tag/v2.3.0-RC11).

File: gulpfile.js [hunk 2]
```diff
@@ -326,10 +326,10 @@ function replaceVersionNumbers() {
...
     .pipe(replace('[android.latest-os-version]', '11'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
-    .pipe(replace('[android.current-app-version]', '2.2.1'))
+    .pipe(replace('[android.current-app-version]', '2.3.0'))
```
Comment (Ein-Tim) [inline gulpfile.js:332]: Latest Android 2.3 RC is [2.3.2-RC2](https://github.com/corona-warn-app/cwa-app-android/releases/tag/v2.3.2-RC2).

**Review by Ein-Tim — APPROVED** [review]
The last updated time is the 02.06. and not today, but as long as it's not in the future 😁

**Comment by dsarkar** [comment]
@Ein-Tim Thanks for the comments. Will fix. As usual: these dates, version numbers might still change again!

**Comment by Ein-Tim** [comment]
Version 2.3 is released under iOS. We should merge this now.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

> Version 2.3 is released under iOS

So the App Store https://apps.apple.com/de/app/corona-warn-app/id1512595757 is at 2.3.2 and https://github.com/corona-warn-app/cwa-app-ios/releases shows only release candidates for 2.3.2.

Would be good to hear from the Open Source Team what the plan is to mark releases on GitHub and what will happen with Android 2.3.x.

**Comment by Ein-Tim** [comment]
@MikeMcC399 Right! 

I opened https://github.com/corona-warn-app/cwa-app-ios/issues/2868 & https://github.com/corona-warn-app/cwa-app-android/issues/3380 regarding this.

**Comment by Ein-Tim** [comment]
@d4rken stated in https://github.com/corona-warn-app/cwa-app-android/issues/3380#issuecomment-856984264 that Android is also in staged rollout already. 
We should merge all version 2.3 related PRs now @dsarkar @heinezen.

**Pull request merged**